hydra/.htaccess

```diff
@@ -0,0 +1,7 @@
+
+RewriteEngine On
+RewriteBase /ns/hydra/
+
+RewriteRule ^spec/(.*)$ https://www.hydra-cg.com/spec/$1 [R=302,L]
+RewriteRule ^context\.jsonld$ core [R,L]
+
```

Comment:
As shared over mail already but for convenience here too:

It would be great to add the following here:

```
RewriteRule ^core/context\.jsonld$ core [R,L]

RewriteRule ^core$ core.jsonld [L,PT]
RewriteRule ^error$ error.jsonld [L,PT]
```

and make sure that

```
Header always set Access-Control-Allow-Origin "*"
```

is set